syntax_and_semantics/literals/string.md

```diff
@@ -61,7 +61,14 @@ A string literal with interpolation allows to embed expressions into the string
 ```crystal
 a = 1
 b = 2
-"sum: #{a} + #{b} = #{a + b}"  # => "sum: 1 + 2 = 3"
+"sum: #{a} + #{b} = #{a + b}" # => "sum: 1 + 2 = 3"
+```
+
+There are also two other types of interpolation using `Kernel#sprintf`. They are not expanded at runtime like the other one. See [String#%](https://crystal-lang.org/api/master/String.html#%25(other)-instance-method).
```

Comment:
I think you mean the opposite: these *are* expanded at runtime, the other one isn't.

Reply:
But technically, both `Kernel.sprintf` and `String#%` are just API methods provided by the stdlib and not really relevant in this section about string literals.
It might be helpful to add a short note about this, but I wouldn't expand it to much.

Reply:
You are right. I think I just take out the sprintf-interpolations and just add the examples to https://crystal-lang.org/api/master/String.html#%25(other)-instance-method and we just merge the other little improvements.